pages/citizen/registration.py

```diff
@@ -14,9 +14,13 @@ class ID:
         txtMobileNumber = "input#person-phone"
         btnLogin = "button#login-submit-action"
         btnProfile = "#header-profile"
+        lblMobileNumber = "//label[contains(text(), 'Mobile Number')]"
+        lblRequired = "xpath=//label[@for='person-phone']/following-sibling::div[last()]"
```

Comment:
This should be `lblMobileNumberErrrorMessage`

Reply:
Done